### DISCUSSION TITLE
soma workflow failed on UTM

### DISCUSSION THREAD

**Discussion by lmyOmaha** [brainvisa/brainvisa.github.io#188] [Q&A]
Hi,
I need a help for brainVisa using. I have a Macbook Pro M1 Max and running UTM
every time when I run brainVisa, there is always an error on state Sulci recognition, no mater ANN, SAPM or CNN. The graph_version I chose is 3.2 and the default option is Sulci recognition with Deep CNN. Can someone help? Thanks!
![Screenshot 2024-10-01 at 3 24 54 PM](https://github.com/user-attachments/assets/1f6832e4-7f48-4d32-99e0-a885e473918e)

**Comment by denisri** [comment]
Hi,
We are not using Macs, so have not tested the full pipeline execution, actually.
The error you have is not clear enough, we'll add a few details in the message in this situation for the next version.
You may run the command:
```
soma_workflow_gui --database /tmp/soma_workflow_yjuuq51_/soma-workflow.db
```
(or with a directory given in the error message if you re-run it)
and look at the workflow jobs, you may access the standard output and error output of the failed job. Then we have a better chance to understand a bit more what happened.
Denis
